MobileConnectionLibrary/MobileConnectionLibrary/Operator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MobileConnectionLibrary;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace MobileConnection
+{
+    public class Operator : IEnumerable<Subscriber>
+    {
+        public OrganizationName OrganizationName;
+        public int OrganizationTaxIdentificationNumber;
+
+        List<Subscriber> subscriber_list;
```

Comment:
Лучше имя без подчерков (стиль camelCase)
```suggestion
        List<Subscriber> subscriberList;
```